docs/src/architecture/08_concepts/catalyst_docs/proposal.md

```diff
@@ -60,6 +60,21 @@ A list of used [Catalyst Signed Document protected header fields](./../signed_do
   which [`type`](./../signed_doc/spec.md#type) must be equal to
   `48c20109-362a-4d32-9bba-e0a9cf8b45be` value.
 
+#### Fund 14 defined category ids
+
+* id: `0194d490-30bf-7473-81c8-a0eaef369619`
```

Comment:
there should be version too or we can say it is the same

Reply:
No no, for categories we are not going to have an actual document itself, for now we just need a defined ids, so version I think is redundant here.